aeppl/transforms.py

```diff
@@ -205,6 +213,180 @@ def apply(self, fgraph: FunctionGraph):
         return self.default_transform_opt.optimize(fgraph)
 
 
+class MeasurableTransform(Elemwise):
+    """A placeholder used to specify a log-likelihood for a transformed RV"""
```

Comment:
Do you mean "measurable variable" when you say "RV"?  We need to sort out our terminology, because we tend to use "measurable variable", "random variable", and `RandomVariable` interchangeably, but there are some important and subtle differences.

Reply:
Measurable variable is correct here, although our measurable variables are ultimately RandomVariables, just not atomic ones.

Reply:
> Measurable variable is correct here, although our measurable variables are ultimately RandomVariables, just not atomic ones.

It should be the other way around in our typing: i.e. `issubclass(RandomVariable, MeasureVariable) is True`.

Reply:
And it is. I was thinking in terms of random generation behavior, in the sense that (5 + RV) is an RV